tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Update Description 5322.feature

```diff
@@ -84,3 +88,13 @@ Scenario: 9. Service Failure
         | Description         |
         | Another Description |
     Then a response with status code 500 is returned
+
+@5322
+Scenario: 10. Update order description to 100 characters should be successful
+    When the user makes a request to update the description with the ID C000014-02
+        | Description              |
+        | #A string of length 100# |
+    Then a response with status code 204 is returned
+    And the order description for order with id C000014-02 is set to
+        | Description              |
+        | #A string of length 100# |
```

Comment:
new line

Reply:
I take it a test for over 100 chars is in the validation test?

Reply:
Should already be a test in this file for 101.